### PR TITLE
Increment pg container version to 10.16 from 10.15.

### LINKED ISSUE #120: There is a new release of the helm chart that includes community support for OCP
AC:
- [ ] Postgres is updated to 10.15
- [ ] Release includes community support for OCP

### PR DESCRIPTION
**Pull request by erekgit**
### What does this PR do?
Increment pg container version to 10.16 from 10.15.

New release changelog: https://www.postgresql.org/docs/10/release-10-16.html

### What ticket does this PR close?
Resolves #120 ++ : N/A

#### Change log
- [x ] The CHANGELOG has been updated, or
- [ ] This PR does not include user-facing changes and doesn't require a CHANGELOG update

#### Test coverage
- [ ] This PR includes new unit and integration tests to go with the code changes, or
- [x ] The changes in this PR do not require tests

#### Documentation
- [ x] Docs (e.g. `README`s) were updated in this PR, and/or there is a follow-on issue to update docs, or
- [ ] This PR does not require updating any documentation

### CODE REVIEW

**Comment by izgeri** [comment]
Hey @erekgit - thanks for submitting this. I'll get one of the maintainers to take a look to see if they have any feedback before we approve.

Please note that before we can accept this PR, you will need to amend your commits to add the DCO signoff. We have instructions for how to amend your commits to add the signoff [here](https://github.com/cyberark/community/blob/master/CONTRIBUTING.md#signing-off-a-commit).

You might also consider rebasing to compress your changes into one commit - we have instructions for that [here](https://github.com/cyberark/community/blob/master/Conjur/conventions/git-tips-and-tricks.md#cleaning-up-your-commit-history), if you are interested.

File: CHANGELOG.md
Comment (izgeri) [inline CHANGELOG.md:9]:
```suggestion
## [v2.0.3] - 2020-12-30
```

File: CHANGELOG.md
Comment (izgeri) [inline CHANGELOG.md:8]:
```suggestion

## Changed
- The default Postgres server version is incremented to 10.16 from 10.15. [Postgres 10.16](https://www.postgresql.org/docs/10/release-10-16.html)
   [cyberark/conjur-oss-helm-chart#147](https://github.com/cyberark/conjur-oss-helm-chart/pull/147)

```
The changelog entry can go above the line of the last release, in the "Unreleased" section

File: CHANGELOG.md
Comment (izgeri) [inline CHANGELOG.md:17]:
```suggestion
```
I've moved this up, so we can remove it from here.

**Review by izgeri — CHANGES_REQUESTED** [review]
Some minor changes to the CHANGELOG - I can address these after you amend your commits to add signoff, if you prefer.

**Comment by BradleyBoutcher** [comment]
This looks good to me, but I think the commit history definitely needs to be squashed

**Comment by erekgit** [comment]
Even with the Sign Off, the commit isn't showing up as verified:  "Signed-off-by: Erik Anderson <erek.anderson@gmail.com>"

**Comment by erekgit** [comment]
Is this acceptable?

**Comment by izgeri** [comment]
When I look at the commit message, I see:
```
 Postgres 10.16 Update

Signed-off-by: Erik Anderson <erek.anderson@gmail.com>
```

this is fine. we are checking on one more thing, then we will kick off a pipeline build before approving.

**Comment by izgeri** [comment]
@erekgit can you please rebase against the `master` branch of the upstream project? We just updated our gh action so that it should run on your PR - we'd like to just get some test validation before we merge.

**Comment by erekgit** [comment]
@izgeri can you check now? ought to be rebased from  upstream https://github.com/cyberark/conjur-oss-helm-chart.git
 now

**Comment by diverdane** [comment]
@izgeri this change passes Jenkins pipeline builds.

**Review by izgeri — APPROVED** [review]
LGTM. Thanks for submitting this @erekgit!

**Pull request merged**
